notebooks/en/rag_with_hugging_face_gemma_mongodb.ipynb

```diff
@@ -0,0 +1,4011 @@
+{
```

Comment:
<p>"<em>Note: Don’t forget to whitelist the IP for the Python host or 0.0.0.0/0 for any IP when creating proof of concepts.</em> " - This is a super useful note!!! It took me a while to figure you when I was experimenting with MongoDB Atlas for the first time :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I made this mistake a few times as well.